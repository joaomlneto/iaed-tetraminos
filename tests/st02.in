10
3 4 O
3 2 O
3 8 Y
2 5 R
1 7 V
5 3 R
1 2 G
2 6 R
1 6 Y
3 8 O
