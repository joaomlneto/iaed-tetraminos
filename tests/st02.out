9
0

|       OO | 9
|        OO| 8
| GGGGYYYY | 7
|  RRRRR   | 6
|  R  RR   | 5
| OO RR    | 4
|  OORRVVVV| 3
|   OO  YY | 2
|    OO  YY| 1
 ----------
